packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -39,7 +47,14 @@ function EnableNotification({
   source = NotificationPromptSource.NotificationList,
   parentCommentAuthorName,
 }: EnableNotificationProps): ReactElement {
+  const isExtension = !!process.env.TARGET_BROWSER;
```

Comment:
I think we have a utility function for this

Reply:
I instead utilized the `app` variable passed to the boot provider as it is much more accurate 🚢